src/main.rs

```diff
@@ -29,8 +25,13 @@ fn handle_request(mut req: Request<Body>) -> Result<Response<Body>, Error> {
 
     // Pattern match on the request method and path.
     match (req.method(), req.uri().path()) {
-        // If request is a `GET` to the `/` path, send to a named backend
-        (&Method::GET, "/") => {
+        // If request is a `GET` to the `/` path, send a default response.
+        (&Method::GET, "/") => Ok(Response::builder()
+            .status(StatusCode::OK)
+            .body(Body::try_from("Welcome to Fastly Compute@Edge!")?)?),
```

Comment:
What should this say? Is this ok? Other options:
- Success!
- It works!
- You have successfully deployed the starter kit 
- etc

Thoughts / suggestions welcome

Reply:
I think that's fine. Other suggestions, 

"Congratulations!  Welcome to Compute@Edge" 
"Hello, from your new WASM program at Compute@Edge!"
"Success.  Your new WASM programing is running on Compute@Edge!"

Reply:
I think the original proposal is great 👍

Reply:
Ok am going to stick with original. Thanks for the other suggestions.